realgeeks.com.website_and_crm.json

```diff
@@ -0,0 +1,69 @@
+{
+  "providerId": "realgeeks.com",
+  "providerName": "Real Geeks",
+  "serviceId": "realgeeks",
+  "serviceName": "Real Geeks Real Estate Websites",
+  "version": 1,
+  "logoUrl": "https://www.realgeeks.com/static/img/logo_2x.png",
+  "description": "Enables a domain to be used for a Real Geeks website and CRM",
+  "variableDescription": "Self explanatory in the variable names.",
+  "records": [
+    {
+      "groupId": "Website",
+      "type": "APEXCNAME",
+      "host": "@",
+      "pointsTo": "%PRODHOST%",
+      "ttl": 600
+    },
+    {
+      "groupId": "Website",
+      "type": "CNAME",
+      "host": "www",
+      "pointsTo": "ssl.realgeeks.com",
+      "ttl": 600
+    },
+    {
+      "groupId": "CRM",
+      "type": "MX",
+      "host": "rg-mail.www",
+      "pointsTo": "mxa.mailgun.org",
+      "priority": 10,
+      "ttl": 300
+    },
+    {
+      "groupId": "CRM",
+      "type": "MX",
+      "host": "rg-mail.www",
+      "pointsTo": "mxb.mailgun.org",
+      "priority": 10,
+      "ttl": 300
+    },
+    {
+      "groupId": "CRM",
+      "type": "SPFM",
+      "host": "rg-mail.www",
+      "spfRules": "v=spf1 include:mailgun.org ~all"
+    },
```

Comment:
With SPFM, you don't want the v=spf1 or ~all. See 6.10.3 in the spec

Reply:
Ahh, thank you!